regression/iterate_data.py

```diff
@@ -30,12 +34,13 @@ def parse_args() -> tuple[Namespace, dict[str, str]]:
                       type=int,
                       default=2,
                       help='Number of epochs to iterate over the data')
-    args.add_argument('--validate-files', default=False, action='store_true', help='Verify files')
-    args.add_argument('--sample-order-file', type=str, help='File location to store sample order')
-    args.add_argument('--validate-iter-time',
+    args.add_argument('--validate-num-files',
                       default=False,
                       action='store_true',
-                      help='Test iter time')
+                      help='Verify number of dataset files')
+    args.add_argument('--sample-order-file', type=str, help='File location to store sample order')
```

Comment:
when we automatically run this test, is there a default argument to set here or will CORGI handle that for us?

Reply:
CORGI provides the file path.